guessing-game-v2/hardhat.config.js

```diff
@@ -5,4 +5,10 @@ module.exports = {
     sources: "./contracts",
     artifacts: "./src/assets/contract/artifacts"
   },
+  networks: {
+    ten_testnet: {
+      url: "https://testnet.obscu.ro/v1/?u=<key>",
```

Comment:
Don't believe this works with hardhat, you need to use https://testnet.obscu.ro/v1/ endpoint (hardhat doesn't work with query params). Also it's better to externalise the user>id into a .env file which is included in the .gitignore file (otherwise the potential to end up committing it back into git is too risky). 

Have a look how it's done in the current samples repo e.g. https://github.com/ten-protocol/sample-applications/blob/main/number-guessing-game/hardhat.config.js#L6